modules/avatar/ImageAvatar.tsx

```diff
@@ -39,7 +39,7 @@ type IProps = IImageAvatarProps & WithStylesProps;
 const ImageAvatar: React.FC<IProps> = ({
   circleClassName,
   className,
-  backgroundColor,
+  backgroundColor = "#cccccc",
```

Comment:
I think we should bring this default to the ddm, is it possible? it can still bee here , but needs to be ovewritten by the default ddm color if not set.

I think it would be easier if you set the color here from the theme. And then it will reuse ddm's theme color and it would be fine.

Reply:
on ddm this component only extends the baseImageAvatar, does not accept it's own props. Can still change that if you want

Reply:
nvm , missunderstood, I'll change it

Reply:
changed, can you check if it's as you've suggested? @nunoliveira12  @acaldas